### PR TITLE
revise docs for releasing OpenYurt v0.7.0

### PR DESCRIPTION
**Pull request by rambohe-ch**
/kind documentation

#### What this PR does / why we need it:
revise readme and changeLog for OpenYurt v0.7.0

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: README.zh.md [hunk 1]
```diff
@@ -61,22 +62,21 @@ OpenYurt 遵循经典的云边一体化架构。
...
 为了用户更方便的使用OpenYurt，用户可以根据下表，选择合适的方法来安装OpenYurt。
 
 | 体验方法                                 | 参考链接                                                     | 预计时间 |
 | ---------------------------------------- | ---------------------- | -------- |
 | 通过OpenYurt体验中心 | [OpenYurt体验中心](https://openyurt.io/docs/next/installation/openyurt-experience-center/overview) | < 1min   |
-| 从零安装一个OpenYurt集群                 | [yurtctl](https://openyurt.io/docs/next/installation/yurtctl-init-join) | <5min    |
-| 通过Operator把K8s集群转换为OpenYurt集群        | [yurtcluster-operator](https://openyurt.io/docs/next/installation/yurtcluster)       | <5min    |
-| 手动转换K8s集群为OpenYurt集群            | [manual](https://openyurt.io/docs/next/installation/manually-setup)    | 10min   |
+| 从零安装一个OpenYurt集群                 | [yurtadm init/join](https://openyurt.io/zh/docs/next/installation/yurtadm-init-join) | <5min    |
+| 基于Kubernetes手动安装OpenYurt集群            | [manual](https://openyurt.io/docs/next/installation/manually-setup)    | 10min   |
```
Comment (huiwq1990) [inline README.zh.md:73]: We should add a kubernetes prepare step. And review the pr `https://github.com/openyurtio/openyurt.io/pull/109`.

Comment (rambohe-ch) [reply README.zh.md:73]: fixed

**Pull request merged**